### PR TITLE
Spain provinces

### PR DESCRIPTION
**Pull request by paololim**

### CODE REVIEW

File: src/main/scala/io/flow/reference/data/Provinces.scala [hunk 1]
```diff
@@ -56,6 +56,59 @@ object Provinces {
...
+  val EspCr = Province(id = "ESP-CR", iso31662 = "CR", name = "Ciudad Real", country = "ESP", provinceType = io.flow.reference.v0.models.ProvinceType("province"), translations = Some(Seq(io.flow.reference.v0.models.LocalizedTranslation(locale = io.flow.reference.v0.models.Locale(id = "en-US", name = "English - United States", country = "USA", language = "en", numbers = io.flow.reference.v0.models.LocaleNumbers(decimal = ".", group = ",")), name = "Ciudad Real"))))
+  val EspCs = Province(id = "ESP-CS", iso31662 = "CS", name = "Castellon", country = "ESP", provinceType = io.flow.reference.v0.models.ProvinceType("province"), translations = Some(Seq(io.flow.reference.v0.models.LocalizedTranslation(locale = io.flow.reference.v0.models.Locale(id = "en-US", name = "English - United States", country = "USA", language = "en", numbers = io.flow.reference.v0.models.LocaleNumbers(decimal = ".", group = ",")), name = "Castellon"))))
```
Comment (ericluria) [inline src/main/scala/io/flow/reference/data/Provinces.scala]: missing an accent on the `o`

File: src/main/scala/io/flow/reference/data/Provinces.scala [hunk 2]
```diff
@@ -56,6 +56,59 @@ object Provinces {
...
+  val EspC = Province(id = "ESP-C", iso31662 = "C", name = "A Coruña", country = "ESP", provinceType = io.flow.reference.v0.models.ProvinceType("province"), translations = Some(Seq(io.flow.reference.v0.models.LocalizedTranslation(locale = io.flow.reference.v0.models.Locale(id = "en-US", name = "English - United States", country = "USA", language = "en", numbers = io.flow.reference.v0.models.LocaleNumbers(decimal = ".", group = ",")), name = "A Coruña"))))
+  val EspCa = Province(id = "ESP-CA", iso31662 = "CA", name = "Cadiz", country = "ESP", provinceType = io.flow.reference.v0.models.ProvinceType("province"), translations = Some(Seq(io.flow.reference.v0.models.LocalizedTranslation(locale = io.flow.reference.v0.models.Locale(id = "en-US", name = "English - United States", country = "USA", language = "en", numbers = io.flow.reference.v0.models.LocaleNumbers(decimal = ".", group = ",")), name = "Cadiz"))))
```
Comment (ericluria) [inline src/main/scala/io/flow/reference/data/Provinces.scala]: missing accent on the `a`

File: src/main/scala/io/flow/reference/data/Provinces.scala [hunk 3]
```diff
@@ -56,6 +56,59 @@ object Provinces {
...
   val Chn65 = Province(id = "CHN-65", iso31662 = "65", name = "Xinjiang", country = "CHN", provinceType = io.flow.reference.v0.models.ProvinceType("other"), translations = Some(Seq(io.flow.reference.v0.models.LocalizedTranslation(locale = io.flow.reference.v0.models.Locale(id = "en-US", name = "English - United States", country = "USA", language = "en", numbers = io.flow.reference.v0.models.LocaleNumbers(decimal = ".", group = ",")), name = "Xinjiang"), io.flow.reference.v0.models.LocalizedTranslation(locale = io.flow.reference.v0.models.Locale(id = "zh-CN", name = "Chinese - China", country = "CHN", language = "zh", numbers = io.flow.reference.v0.models.LocaleNumbers(decimal = ".", group = ",")), name = "新疆维吾尔自治区"))))
+  val EspA = Province(id = "ESP-A", iso31662 = "A", name = "Alicante", country = "ESP", provinceType = io.flow.reference.v0.models.ProvinceType("province"), translations = Some(Seq(io.flow.reference.v0.models.LocalizedTranslation(locale = io.flow.reference.v0.models.Locale(id = "en-US", name = "English - United States", country = "USA", language = "en", numbers = io.flow.reference.v0.models.LocaleNumbers(decimal = ".", group = ",")), name = "Alicante"))))
```
Comment (ericluria) [inline src/main/scala/io/flow/reference/data/Provinces.scala]: Are the locales correct on all of these?

**Comment by paololim** [comment]
https://github.com/flowcommerce/json-reference/pull/62

**Pull request closed without merging**